FAQ.md

```diff
@@ -0,0 +1,113 @@
+---
+
+# FAQ
+
+### How can I try 10101?
+
+You can try out the public beta immediately by downloading it from [Google Play](https://play.google.com/store/apps/details?id=finance.get10101.app) if you're using an Android smartphone, and from [TestFlight](https://testflight.apple.com/join/WhwnPUh8) if you're using an iPhone.
+
+10101 is a mobile application. It is not possible to access it from your browser.
+### Why "10101"?
+
+10101 is pronounced "ten ten one", and it means 21 in binary. Just a pun!
+
+### What is 10101
+
+It's an all-in-one Bitcoin application that includes a Bitcoin wallet (On-chain) and a Lightning wallet as well, thus making it possible to send and receive bitcoins using both networks.
+
+Once your Lightning wallet has been funded and your channel opened, the application will give you the ability to trade financial derivatives directly from your wallet, and even hold synthetic cash, such as USDp, the first USD on Lightning.
+
+### On-chain? Lightning?
+
+On-chain is a term used to designate the traditional Bitcoin network : It is used for its high level of security and verifiability, and allows large amounts of money to be sent and received for a fee to register transactions in the blockchain.
+
+Lightning is a payment network built on top of Bitcoin (referred as layer), designed to enable instant and practically free transactions once your node and channel are set up, while benefiting from the security of Bitcoin's legacy network.
+
+### Will 10101 manage the funds for me?
+
+No, at no time will 10101 hold your bitcoins.
+
+Our application makes a point of respecting and educating users in the principles of self-custody : you hold your own keys, you remain responsible for your wallet and funds, and you retain full control throughout your experience.
+
+That's why you need to make sure you back up your recovery phrase and keep it safe.
+
+### How is it possible to trade directly from my wallet, without a platform?
+
+This prowess is made possible by Discreet Log Contracts.
+
+These are peer-to-peer contacts, involving two parties who lock funds into a specific channel. The release of which is conditional: when the contract expires, it releases the funds in the proportions corresponding to the conditions defined at the outset, using data from an oracle to ensure that the transaction proceeds smoothly, in order to prevent any non-collaborative outcome.
+
+
+DLCs can be seen as an additional layer built on top of the Lightning Network.
+
+If you want to learn more, we produced a series of [blog posts](https://10101.finance/blog/dlc-to-lightning-part-1) on this topic, and another one on the specific case of [trading settlements](https://10101.finance/blog/trade-settlement).
+
+
+### How do USDp and synthetic currencies work?
+
+USDp and synthetic currencies within 10101 are referred to synthetics in finance.
+You will have a USD balance circulating on Lightning, holding nothing but sats and a short position combined.
```

Comment:
I think this is _how_ it works, but maybe you want to add something else. Maybe an example, but I wasn't sure how to construct it.

```suggestion
USDp represents a stable USD balance because it is backed by a DLC which pays out a variable number of sats corresponding to a _fixed_ amount of USD.
```